modules/openapi-generator/src/main/resources/python/api_client.mustache

```diff
@@ -391,10 +394,7 @@ class ApiClient:
         """
 
         # fetch data from response object
-        try:
-            data = json.loads(response_text)
-        except ValueError:
-            data = response_text
```

Comment:
I have removed this exception handling, but I am not sure how it affects you.
This change would help find bugs like #17987.

Reply:
This is not going to work, so either undo it or I'll come up with a different approach.

Reply:
This change has been reverted.